Documentation/helm-operator.md

```diff
@@ -101,7 +101,9 @@ The following tables lists the configurable parameters of the rook-operator char
 | `csi.enableCephfsDriver`           | Enable Ceph CSI CephFS driver.                                                                                                 | `true`                                                    |
 | `csi.pluginPriorityClassName`      | PriorityClassName to be set on csi driver plugin pods.                                                                         | <none>                                                    |
 | `csi.provisionerPriorityClassName` | PriorityClassName to be set on csi driver provisioner pods.                                                                    | <none>                                                    |
-| `csi.enableOMAPGenerator`          | EnableOMAP generator deploys omap sidecar in CSI provisioner pod, to enable it set it to true                                  | `false`                                                    |
+| `csi.enableOMAPGenerator`          | EnableOMAP generator deploys omap sidecar in CSI provisioner pod, to enable it set it to true                                  | `false`                                                   |
```

Comment:
Since we only document the csi settings in the helm doc, there are a lot of csi settings that may be interesting to the user and they might not find them. How about adding a link from the ceph-csi-drivers.md to this helm doc where they can find the settings? Thought it could be a separate PR if you want.

Reply:
I can send a separate PR to improve the default and available configuration for CSI.

Reply:
Is there a link to explain what omap does for users?

Also, does omap need to be capitalized (i.e., OMAP)?